OPERATING.md

```diff
@@ -1,14 +1,14 @@
-# Operating Guide: Data Transformation and Ingest for VEDA
+# Operating Guide: Data Transformation and Ingest for MAAP
 
-This guide provides information on how VEDA runs data ingest, transformation and metadata (STAC) publication workflows via AWS Services, such as step functions.
+This guide provides information on how MAAP runs data ingest, transformation and metadata (STAC) publication workflows via AWS Services, such as step functions.
 
 NOTE: Since collection ingest still requires calling the database from a local machine, users must add their IP to an inbound rule on the security group attached to the RDS instance.
```

Comment:
We change change this section now that we're using the `/collections` endpoint

Reply:
Thanks @jjfrench, just removed the note.